nixos/modules/hardware/xone.nix

```diff
@@ -21,6 +21,7 @@ in
       extraModulePackages = with config.boot.kernelPackages; [ xone ];
     };
     hardware.firmware = [ pkgs.xow_dongle-firmware ];
+    hardware.xpad-noone.enable = lib.mkDefault true;
```

Comment:
Not sure if this is a good idea...

Reply:
Since xpad-noone was mentioned in the PR of xone's init, I added it. I hope it won't bring any unexpected side effects.

Reply:
I think this change deserves its own PR with a release notes entry

Reply:
> I think this change deserves its own PR with a release notes entry

ok, I have dropped the commit. I will create a new PR while the PR being merged